R/pipgd_pov.R

```diff
@@ -358,7 +358,8 @@ pipgd_pov_severity_nv <- function(
     #   Computations -----------------------------------------------------------
 
     if (!is.null(pov_gap)) {
-      if (is.null(pov_gap$pov_stats$pov_gap)) {
+      if (pov_gap != pipgd_pov_gap_nv(welfare = welfare, weight = weight)$pov_stats$pov_gap) {
```

Comment:
@zander-prinsloo, I think this condition is unnecessary. I understand the logic of making sure everything is consistent, but it will drop down dramatically the efficiency of the function.